jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanManager.kt

```diff
@@ -110,6 +110,7 @@ internal class CodeWhispererCodeScanManager(val project: Project) {
         var codeScanStatus: Result = Result.Failed
         val startTime = Instant.now().toEpochMilli()
         var codeScanResponseContext = defaultCodeScanResponseContext()
+        var projectSize = 0L
```

Comment:
nit: the telemetry field is optional, for unsupported language or no workspaceFolder case (not sure if applicable to JB), simply don't set it instead of `0`

Reply:
Set to null by default.